### DISCUSSION TITLE
Unable to fetch the example Datasets

### DISCUSSION THREAD

**Discussion by joaquingabad** [GAA-UAM/scikit-fda#321] [Q&A] [answered]
Hello!

I am really interested in using your library scikit-fda. I am trying to launch the examples to see how to implement the functions and the shape of input data, but an error is triggered when i try it. I do not know what the problem is. I have tried reinstall the package, also rdata and scikit-datasets.

I will really apreciatte your help on this. Tank you!

![Captura de pantalla de 2021-02-11 17-16-53](https://user-images.githubusercontent.com/63837405/107664688-fa919c00-6c8c-11eb-991c-cda4ca69df82.png)

**Comment by vnmabus** [comment, marked as answer]
Are you sure `scikit-datasets` is installed? What do you get as the output of `pip list`?

**Reply by joaquingabad** [reply]
Yes it is installed in his version 0.1.38. I don't know what is wrong.

![Captura de pantalla de 2021-02-11 18-54-14](https://user-images.githubusercontent.com/63837405/107677487-9c6bb580-6c9a-11eb-883a-e9ddd930e51f.png)

**Reply by vnmabus** [reply]
This is really strange. Can you import `skdatasets` directly from the interpreter?

**Reply by joaquingabad** [reply]
When i do it returns the error

![Captura de pantalla de 2021-02-12 08-10-06](https://user-images.githubusercontent.com/63837405/107739709-c7412280-6d09-11eb-978f-5bdca50287fc.png)

**Reply by vnmabus** [reply]
So, a few basic questions:

- I see that you are using Jupyter. Are you sure that you are using the kernel that has `scikit-datasets` installed? Maybe you should try `!pip list` inside Jupyter, just in case.
- Have you tried reinstalling `scikit-datasets`? Uninstalling and then installing again? Installing an older version?
- What content does it have the site-packages folder of your installation? Is `skdataset` a subfolder?
- What version of Python are you using? What Operating System?

**Reply by joaquingabad** [reply]
Exactly! reinstall scikit-datasets has solved the problem. I am working in Ubuntu with Python 3.8.3 version and everything is working properly now.
Tank you so much for your work, the resource and your close help.
Muchas gracias!!

**Reply by vnmabus** [reply]
De nada. I hope that the library is useful for you, and do not hesitate to ask again if you have any other issue 😉.
